src/dataflow/source/kafka.rs

```diff
@@ -603,6 +623,9 @@ where
                 // Accumulate updates to BYTES_READ_COUNTER;
                 let mut bytes_read = 0;
                 if let Some(mut dp_info) = dp_info.as_mut() {
+                    KAFKA_SOURCE_OPERATOR_SCHEDULED_COUNTER
+                        .with_label_values(&[&dp_info.topic_name, &dp_info.source_id])
+                        .inc();
```

Comment:
Just to check, this is the same number that comes out of `mz_scheduling_histogram`?

Reply:
i think so yeah

Reply:
This will only get scheduled when the operator for the "active" source (the one actually reading the data) gets scheduled. Is that intentional?

Reply:
yup -- didn't want to pollute the metrics for the other workers where its essentially a no-op